pkg/storage/utils/decomposedfs/upload.go

```diff
@@ -360,6 +358,33 @@ func (fs *Decomposedfs) GetUpload(ctx context.Context, id string) (tusd.Upload,
 	}, nil
 }
 
+// lookupNode looks up nodes by path.
+// This method can also handle lookups for paths which contain chunking information.
```

Comment:
could you add a reason how this function differs from the one in the lookup interface? Do you think other implementations of the decomposedfs need the same handling / detection of chunked paths? should `fs.lu.NodeFromPath` maybe get a parameter to handle chunked paths? why not add a `fs.lu.NodeFromChunkedPath` and refactor the two functions to share more code?

Reply:
Ah, now I see that you are calling `fs.lu.NodeFromPath` in `lookupNode` ... nevermind my comment ;-)